### PR TITLE
24 jordys individual part

### LINKED ISSUE #24: Jordy's individual part

### PR DESCRIPTION
**Pull request by rafaeltab**
Closes #24

### CODE REVIEW

File: Verslag/individual/jordy/Description.tex [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+The current official version of Mozilla Firefox is 94.1.2.
+To get the official Firefox version 85.1.3 it was necessary to download the application from another source.
+The APK file was checked with \href{https://www.virustotal.com/gui/file/59ce0f9ea256b4576f391d01c685ced2db224a252bf09c3f362e6859a6c7ead5/details}{VirusTotal} and compared to the most recent official version of Mozilla Firefox.
+This process has been described in detail in chapter 4.1.
+
+The official APK file for Mozilla Firefox version 85.1.3 had the package name “org.mozilla.firefox” and had the size of 63.39 MB.
+All aspects were relatively similar been the versions 85.1.3 and 94.1.2 of Mozilla Firefox with a few small alterations.
+These alterations were because one of the versions was a few months older.
+
+From this moment onwards in the chapter the “official application” will be referring to the 85.1.3 version of Mozilla Firefox with the package name “org.mozilla.firefox” and the “malicious application” will be referring to APK file found on Koodous with the package name “com.eset.ems2.gp”.
```
Comment (rafaeltab) [inline Verslag/individual/jordy/Description.tex]: to the APK file found

File: Verslag/individual/jordy/VirusTotal summary.tex [hunk 2]
```diff
@@ -0,0 +1,202 @@
+\subsection{VirusTotal summary}
+The selected malicious application was checked on VirusTotal for a first overview of the application.
```
Comment (rafaeltab) [inline Verslag/individual/jordy/VirusTotal summary.tex]: first overview sounds weird
possible fixes:
`first impression`
`an overview`

File: Verslag/individual/jordy/VirusTotal summary.tex [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+    NANO-Antivirus           & Trojan.Android.Hidden.estanm         \\ \hline
+    Sangfor Engine Zero      & Malware.Android-Script.Save.dc318d22 \\ \hline
+    Sophos                   & Andr/Xgen2-EQ                        \\ \hline
+    Symantec Mobile Insight  & Trojan:Malapp                        \\ \hline
+    Trustlook                & Android.Malware.General (score:9)    \\ \hline
+    Yandex                   & Trojan.Mober.bTTGwd.156              \\ \hline
+    ZoneAlarm by Check Point & HEUR:Trojan-Spy.AndroidOS.Agent.vp   \\ \hline
+\end{tabular}
+
+These flags from the different security vendors clearly indicate that the malicious application is in fact malicious and most likely a Trojan.
```
Comment (rafaeltab) [inline Verslag/individual/jordy/VirusTotal summary.tex:41]: Could use footnote like @Discopizza and @rafaeltab but is not a requirement

File: Verslag/individual/jordy/VirusTotal summary.tex [hunk 4]
```diff
@@ -0,0 +1,202 @@
+\subsection{VirusTotal summary}
+The selected malicious application was checked on VirusTotal for a first overview of the application.
+On this website it is possible to see if security vendors flag this application as malicious, what permissions the application asks for, the package name and many more aspects of the application.
+With this information both of the applications were compared with each other.
+In this chapter the process and results are described.
```
Comment (rafaeltab) [inline Verslag/individual/jordy/VirusTotal summary.tex:5]: were described. Everywhere?

File: Verslag/individual/jordy/VirusTotal summary.tex [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+\newline \texttt{android.permission.READ\_EXTERNAL\_STORAGE}
+\newline \texttt{android.permission.RECEIVE\_BOOT\_COMPLETED}
+\newline \texttt{android.permission.WRITE\_SETTINGS}
+\newline \texttt{android.permission.VIBRATE}
+\newline \texttt{android.permission.KILL\_BACKGROUND\_PROCESSES}
+\newline \texttt{android.permission.WAKE\_LOCK}
+\newline \texttt{android.permission.GET\_ACCOUNTS}
+
+A lot of these permissions are not usually asked for by an internet browser.
+A in depth comparison of the differences between the applications permissions was described in detail in chapter 4.1.2.3.
```
Comment (rafaeltab) [inline Verslag/individual/jordy/VirusTotal summary.tex]: An in depth

File: Verslag/individual/jordy/Behavior analysis.tex [hunk 6]
```diff
@@ -0,0 +1,19 @@
+\subsection{Behavior analysis}
+To get a better understanding of the malicious application a behavior analysis was performed.
```
Comment (rafaeltab) [inline Verslag/individual/jordy/Behavior analysis.tex:2]: this don't read nicely.
maybe `A behavior analysis was performed to get a better understanding of the malicious application`
or add a comma between application and a

File: Verslag/individual/jordy/Behavior analysis.tex [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+The process and results of the behavior analysis are described in this chapter.
+
+When a user installed the application it showed up in the overall app screen.
+The application was installed already.
+When the app was opened it showed an installation screen with all the permissions that were described in chapter 4.1.2.1 along with the button install.
```
Comment (rafaeltab) [inline Verslag/individual/jordy/Behavior analysis.tex:7]: if you add a `\label` in chapter 4.1.2.1 you could use`\ref{}` here to A create a link B not have to change it when the numbers change

File: Verslag/individual/jordy/Network Analysis.tex [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+
+\subsubsection{HTTP proxy analysis}
+Mitmproxy was used to capture the network traffic to and from the emulated phone.
+Mitmproxy ended up showing that the phone made a connection with multiple different IP addresses.
+After around 30 minutes of capturing the network traffic it kept connecting and disconnecting with the same 5 IP addresses.
+A couple of times an IP address came up that was in a completely different range of IP’s.
+
+In the picture that is shown below there are a total of 8 IP addresses that are captured again and again.
+5 out of the 8 IP addresses came from the same 3. range of IP addresses.
+The remaining 3 IP addresses were very different from the 3.  range and they were not consistently looping.
```
Comment (rafaeltab) [inline Verslag/individual/jordy/Network Analysis.tex]: `came from the same 3. range of IP addresses`
make this
`all started with 3`

File: Verslag/individual/jordy/Network Analysis.tex [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+    \centering
+    \includegraphics[width=0.85\textwidth]{MitmwebCapture.png}
+    \caption{Captured network traffic with mitmproxy}
+    \label{jordy-mitmweb}
+\end{figure}
+
+\newpage
+\subsubsection{Wireshark analysis}
+The captured network traffic in mitmproxy indicated that there were multiple connections to different IP addresses.
+As described in subchapter 4.3.1 there was 1 range with 5 different IP addresses that kept looping.
```
Comment (rafaeltab) [inline Verslag/individual/jordy/Network Analysis.tex:26]: again you could use label with ref here

File: Verslag/individual/jordy/Code Analysis.tex [hunk 10]
```diff
@@ -0,0 +1,43 @@
+\subsection{Code analysis}
+To get a better understanding of how the malicious application was build a code analysis was performed using jadx.
+The process and results of the code analysis are described in this chapter.
+
+On first glance it became obvious that the people behind the application obfuscated the code.
+This meant that the code was difficult to understand.
+Because of this it was possible to get an idea of what the code did, but it was not possible to say with certain that it was correct.
+
+In the code it was possible to confirm a lot of the permissions that were described in subchapter 4.1.2.1.
```
Comment (rafaeltab) [inline Verslag/individual/jordy/Code Analysis.tex:9]: Again use ref here

File: Verslag/individual/jordy/Countermeasures and detection.tex [hunk 11]
```diff
@@ -0,0 +1,38 @@
+\subsection{Countermeasures and detection}
+A part of malware analysis is coming up with countermeasures and ways to detect the application.
+The different countermeasures are described in this chapter.
+
+\subsubsection{Preventive measures}
+There would be a few preventive measures to not install this malicious application.
+One of these was to only install applications that are on the official Google Play Store.
+The only Firefox application that is downloadable on the Google Play Store is the official application from Mozilla.
+This would mean that the malicious application is not able to be installed on an user’s phone.
```
Comment (rafaeltab) [inline Verslag/individual/jordy/Countermeasures and detection.tex]: on a user's phone

**Review by rafaeltab — COMMENTED** [review]
Just change a few things

**Review by Wolkje — APPROVED** [review]

**Pull request merged**